script/find_record.py

```diff
@@ -0,0 +1,34 @@
+import json 
+from sys import argv
+
+
+script, JSON_FILE, user_name  = argv
+
+
+#JSON_FILE = "init\persons.json"
+unflat_persons = json.load(open(JSON_FILE, encoding='utf-8'))
+ 
+
+def find_user_name(first_name):
+  list_of_persons = unflat_persons['results']
+  record_found = False
+  user_list = []
+  for dict_of_person in list_of_persons:   
+    gender = dict_of_person['gender']
+    title = dict_of_person['name']['title']
+    first_name = dict_of_person['name']['first']
+    last_name = dict_of_person['name']['last']
+    if first_name == user_name:
+      record_found = True
+      user_list.append((gender, title, first_name, last_name))
+  if record_found == False:
+    print('A user with this name does not exist')
+  return(user_list)
```

Comment:
Why parents here?

Reply:
user_list is a new array of records. I don't understand what is wrong.

Reply:
It called list in python :-)
But why:
```python
return(user_list)
```
instead of just
```python
return user_list
```
?

Reply:
I had to instinctively add parentheses as I add to "print"